cmd/gcp-controller-manager/node_csr_approver.go

```diff
@@ -698,6 +698,29 @@ func clusterHasInstance(ctx *controllerContext, instanceZone string, instanceID
 		return false, err
 	}
 
+	// instanceZone looks like
+	// "https://www.googleapis.com/compute/v1/projects/my-project/zones/us-central1-c"
+	// Extract the bare zone name just "us-central1-c".
+	instanceZoneName := path.Base(instanceZone)
+
+	if ctx.csrApproverUseListReferrers {
+		klog.Infof("using ListReferrers to verify instance cluster membership")
```

Comment:
also log the context: csr.Name, and instance id.
This log might be too noisy? Suggest increasing verbosity here.

Reply:
updated verbosity and added instance.Name to log.